### PR TITLE
Search/controller

### PR DESCRIPTION
**Pull request by maryfranz**
Added Test structure project, copyright, .gitignore file and first Controller class code

### CODE REVIEW

File: src/main/java/com/foundation/search/controller/Controller.java [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+ * This software is the confidential and proprietary information of
+ * Jalasoft, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
 package com.foundation.search.controller;
 
 public class Controller {
```
Comment (emelyllanos) [inline src/main/java/com/foundation/search/controller/Controller.java:20]: } is missing, please add it

File: src/main/java/com/foundation/search/model/Search.java [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+ * This software is the confidential and proprietary information of
+ * Jalasoft, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
 package com.foundation.search.model;
 
 public class Search {
```
Comment (emelyllanos) [inline src/main/java/com/foundation/search/model/Search.java:15]: } is missing. please add it

File: src/main/java/com/foundation/search/view/View.java [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+ * This software is the confidential and proprietary information of
+ * Jalasoft, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
 package com.foundation.search.view;
 
 public class View {
```
Comment (emelyllanos) [inline src/main/java/com/foundation/search/view/View.java:15]: } is missing. Please add it

File: src/main/java/com/foundation/search/controller/Controller.java [hunk 4]
```diff
@@ -12,7 +12,25 @@
...
+        this.view = new View();
+        this.search = new Search();
+    }
+
+    public void getBasicFields() {
+
+    }
+
+    public void findEvent(ActionEvent buttonEvent) {
+
```
Comment (emelyllanos) [inline src/main/java/com/foundation/search/controller/Controller.java:34]: Looks good

File: src/main/java/com/foundation/search/Main.java [hunk 5]
```diff
@@ -12,8 +12,16 @@
...
     public static void main(String [] args) {
         System.out.println("This is the main class");
+        Init();
+    }
+
+    public static void Init() {
+        System.out.println("Controller Init");
+        Controller controller = new Controller();
     }
 }
```
Comment (emelyllanos) [inline src/main/java/com/foundation/search/Main.java:27]: looks good

File: src/main/java/com/foundation/search/model/Search.java (hunk 2 above)
Comment (maryfranz) [reply src/main/java/com/foundation/search/model/Search.java:15]: After taking a look with you, there is not missing }

File: src/main/java/com/foundation/search/view/View.java (hunk 3 above)
Comment (maryfranz) [reply src/main/java/com/foundation/search/view/View.java:15]: Same comment here, there is not missing }

**Review by emelyllanos — APPROVED** [review]
Pleae merge the changes

**Pull request merged**